libnet/sample/Makefile.am

```diff
@@ -77,3 +77,9 @@ test_ipv6_icmpv4_SOURCES = test_ipv6_icmpv4.c
 EXTRA_DIST = libnet_test.h
 
 LDADD 			= $(top_builddir)/src/libnet.la
+
+EXTRA_DIST += arp-new.c win32/arp/arp.vcproj win32/cdp/cdp.vcproj	\
```

Comment:
yeaaaah I must have been drunk when I wrote that. Visual Studio usually leaves a lot of mess and dot files in project directory after you open a .vcproj file. The change I suggested would add all that to the tar ball, which is of course a terrible idea...

Any thoughts regarding the 100+ targets thingy I wrote prior to this?